servers/mu/src/domain/api/pushMsg.js

```diff
@@ -0,0 +1,81 @@
+import { Rejected, Resolved, fromPromise, of } from 'hyper-async'
+
+import { getCuAddressWith } from '../lib/get-cu-address.js'
+import { pullResultWith } from '../lib/pull-result.js'
+import { graphqlReturnSchema } from '../dal.js'
+
+export function pushMsgWith ({
+  selectNode,
+  fetchResult,
+  fetchTransactions,
+  crank,
+  logger,
+}) {
+  const getCuAddress = getCuAddressWith({ selectNode, logger })
+  const pullResult = pullResultWith({ fetchResult, logger })
+  const fetchTransactionsAsync = fromPromise(fetchTransactions)
+
+  return (ctx) => {
+    return of(ctx)
+      .chain((ctx) => {
+        return fetchTransactionsAsync([ctx.tx.id])
+      })
+      .chain(res => {
+        if(res?.data?.transactions?.edges?.length >= 1) {
+          if(res.data.transactions.edges[0].block?.timestamp) {
+            const oneDayAgo = Date.now() - 24 * 60 * 60 * 1000;
```

Comment:
Here you want to use an env var or {BlockHeight == "1572102"}

then compare transaction.edges[0].block?.height > process.env['ALLOW_PUSHES_AFERT'] || "1572085"

Reply:
Ok got it resolved in the latest commit